practice code with R-Pi

laurentallerico opened the pull request:
Please accept my pull request. It's good, I tested it.

laurentallerico committed:
practice code with R-Pi

anthonyjclark reviewed (changes requested):
Please put these in a directory called: `rpi/demos/`

laurentallerico committed:
Update hiLow.py

added comments

laurentallerico committed:
Update ledFade.py

added comments (check that sleep(.05) is actually 4 seconds... that comment was left from when we originally wrote the code)

laurentallerico committed:
Update pwmTest.py

Added comments

laurentallerico committed:
Create temp.txt

laurentallerico committed:
Delete temp.txt

laurentallerico committed:
created a folder rpi_demos

laurentallerico committed:
corrected directory from rpi_demos to rpi/demos/

laurentallerico commented:
Done! let me know if I should edit the comments on the .py files

anthonyjclark reviewed (approved):
Looks great. Thanks!
